vector3/vector3.go

```diff
@@ -330,7 +355,7 @@ func (v Vector[T]) XZY() Vector[T] {
 	}
 }
 
-func (v Vector[T]) ZXY() Vector[T] {
+func (v Vector[T]) ZPosition() Vector[T] {
```

Comment:
Let's keep it ZXY (this common naming convention for swizzles)
https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/dx9-graphics-reference-asm-ps-registers-modifiers-source-register-swizzling

This goes for the other swizzle esc ops as well

Reply:
yep, that was a find and replace operation )) i'll revert )